common-npm-packages/az-blobstorage-provider-v2/package.json

```diff
@@ -14,7 +14,7 @@
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
     "q": "1.4.1",
-    "artifact-engine": "0.1.29",
+    "artifact-engine": "1.0.0",
```

Comment:
What are other changes we introduce with 0.1.29 -> 1.0.0? Are there any risks?

Reply:
The list of changes:

- Upgrade handlebars to resolve vulnerabilities for dependent pipeline tasks
- Upgrade azure-pipelines-task-lib to "^3.1.0" version
- Updated dependencies; Removed unused definitions
- Changed typescript version to 4.0.2;
- Changed deprecated assert.notEqual and assert.equal to assert.notStrictEqual and assert.strictEqual;
- Removed deprecated @types/handlebars reference, since handlebars lib has already build-in type definitions

The library was tested as a part of DownloadBuldArtifactsV0
Also, I've tested JavaToolInstallerV0 task with Azure Storage as JDK source